deploy/helm/sumologic/values.yaml

```diff
@@ -4339,6 +4339,21 @@ metadata:
               - batch
             exporters:
               - sumologic/containers
+          ## This is the same pipeline like for logs/fluent/containers with the following modifications:
+          ## - filter/include_fluent_tag_containers and attributes/remove_fluent_tag are being removed as there are only containers log
+          ## - attributes/containers functionality is being replaced by otellogs operators
+          logs/otlp/containers:
+            receivers:
+              - otlp
+            processors:
+              - memory_limiter
+              - groupbyattrs/containers
+              - k8s_tagger
+              - source/containers
+              - resource/containers_copy_node_to_host
+              - batch
+            exporters:
+              - sumologic/containers
```

Comment:
We could keep backward compatibility with fluent-bit pipeline and merge `logs/otlp/containers` with `logs/fluent/containers`, but not really sure if it's good way